### PR TITLE
[Keymap] Add mly32 keymap for GMMK Pro Ansi

### PR DESCRIPTION
**Pull request by mly32**
## Description

Added a custom layout for GMMK Pro with a custom RGB matrix effect. Details in `readme.md`.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gmmk/pro/ansi/keymaps/mly32/keymap.c [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+#ifdef ENCODER_ENABLE
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (get_mods() & MOD_MASK_CA && !(get_mods() & ~MOD_MASK_CA)) {
+        /* only active mods are LCTRL, RCTRL, LALT, or RALT */
+        tap_code16(clockwise ? KC_TAB : LSFT(KC_TAB));
+    } else {
+        tap_code16(clockwise ? KC_VOLU : KC_VOLD);
+    }
+
+    return true;
```
Comment (drashna) [inline keyboards/gmmk/pro/ansi/keymaps/mly32/keymap.c:50]:
```suggestion
    return false;
```

**Comment by mly32** [comment]
Nice catch. Thank you.

**Review by drashna — APPROVED** [review]

**Comment by mly32** [comment]
^ bump.

File: keyboards/gmmk/pro/ansi/keymaps/mly32/keymap.c [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+#include "binds.c"
```
Comment (fauxpark) [inline keyboards/gmmk/pro/ansi/keymaps/mly32/keymap.c:17]: Don't include .c files in other .c files. Put the keymap array in this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**